### PR TITLE
Regex comma escape added

### PR DESCRIPTION
**Pull request by misantron**
A small fix for AdBlockRule regex build function.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
+/.idea
```
Comment (limonte) [inline .gitignore]: Undo changes in `.gitignore`, your local environment files/folders should be ignored globally: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (misantron) [reply .gitignore]: Sure

**Comment by limonte** [comment]
Perfect! Thanks a lot!

**Comment by limonte** [comment]
https://github.com/limonte/php-adblock-parser/releases/tag/v0.1.5 :tada:

**Pull request merged**